settings/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/SIGBlockchain/project_aurum/internal/config"
+	"github.com/SIGBlockchain/project_aurum/internal/jsonify"
+)
+
+func main() {
+	//open configuration file
+	configFile, err := os.Open("../cmd/config.json")
+	if err != nil {
+		log.Fatal("Failed to open configuration file : " + err.Error())
+	}
+	defer configFile.Close()
+
+	cfg := config.Config{}
+	err = jsonify.LoadJSON(configFile, &cfg)
+	if err != nil {
+		log.Fatal("Failed to unmarshall configuration data : " + err.Error())
+	}
+
+	//specify flags
+	versionU64 := flag.Uint("version", uint(cfg.Version), "enter version number")
+	flag.Uint64Var(&cfg.InitialAurumSupply, "initialaurum", cfg.InitialAurumSupply, "enter a number for initial aurum supply")
+	flag.StringVar(&cfg.Port, "port", cfg.Port, "enter port number")
+	flag.StringVar(&cfg.BlockProductionInterval, "blockproditv", cfg.BlockProductionInterval, "enter a time for block production interval\n(assuming seconds if units are not provided)")
+	flag.BoolVar(&cfg.Localhost, "localhost", cfg.Localhost, "syntax: -localhost=/boolean here/")
+
+	//read flags
+	flag.Parse()
+
+	cfg.Version = uint16(*versionU64)
+	//check block production interval suffix
+	hasSuf := false
+	for _, s := range [7]string{"ns", "us", "µs", "ms", "s", "m", "h"} {
+		if strings.HasSuffix(cfg.BlockProductionInterval, s) {
```

Comment:
Can you include in the instructions that it defaults to seconds? Or a simpler solution would be to just say "invalid suffix" if the suffix doesn't exist in the list here.

Reply:
I've included a hint in the instructions that says that it defaults to seconds.
`33: flag.StringVar(&cfg.BlockProductionInterval, "blockproditv", cfg.BlockProductionInterval, "enter a time for block production interval\n(assuming seconds if units are not provided)")`